app/helpers/document_helper.rb

```diff
@@ -32,4 +40,14 @@ def titled_reference_or_file_name(document)
   def reference_or_file_name(document)
     document.numbers.presence || document.name
   end
+
+  def tags_first_half(tags)
+    count = (tags.count.round / 2.to_f).ceil
+    tags.in_groups_of(count).first
+  end
+
+  def tags_second_half(tags)
+    count = (tags.count.round / 2.to_f).ceil
+    tags.in_groups_of(count).second
+  end
```

Comment:
You don't need these methods - you can just use `in_groups(2, false)` to split the array into two halves (the `false` prevents the second array being filled with `nil` when there's an odd number).